### PR TITLE
Removes the code for the four removed gateways

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
#66161 removed the maps for academy, challenge, spacebattle, and wild west, but left the code in the awaymissions module for them. This PR moves code that's used elsewhere out of them and deletes the files themselves.

## Why It's Good For The Game
Leaving depreciated code from gateways very unlikely to come back is unnecessary is bad and code cleanup is good

## Changelog

:cl:
code: Code for Academy, Challenge, Spacebattle, and Wild West away missions has been removed
/:cl:

### CODE REVIEW

**Comment by Zonespace27** [comment]
Drafted while i sort out map compiles

File: code/modules/awaymissions/mission_code/Cabin.dm [hunk 1]
```diff
@@ -103,6 +103,24 @@
...
+/obj/structure/ladder/unbreakable/rune
+	name = "\improper Teleportation Rune"
+	desc = "Could lead anywhere."
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	color = rgb(0,0,255)
+
+/obj/structure/ladder/unbreakable/rune/ComponentInitialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_blocker)
```
Comment (JohnFulpWillard) [inline code/modules/awaymissions/mission_code/Cabin.dm:115]: can you put this on Initialize() instead? We're trying to move away from ComponentInitialize from what I understand.

Comment (SuperNovaa41) [reply code/modules/awaymissions/mission_code/Cabin.dm:115]: Yes it's deprecated it shouldn't be used

Comment (Zonespace27) [reply code/modules/awaymissions/mission_code/Cabin.dm]: Resolved

**Pull request merged**